build & release fails on resolving 'versions.json'

https://github.com/evanderkoogh/otel-cf-workers/actions/runs/10773791169/job/29874355832

```
Error: [check:type-esm] src/sdk.ts(32,27): error TS2307: Cannot find module '../versions.json' or its corresponding type declarations.
```

Reason appears to be that `npm run build` occurs before `npm run release` in the github action, and if the versions file isn't written already, the build script fails.

```
    // from .github/workflows/release.yml
        run: 'pnpm install --frozen-lockfile --child-concurrency=10'
      - name: Check the build
        run: pnpm build
      - name: 'Check for Errors'
        run: 'pnpm check'
      - name: Create Release Pull Request or Publish to npm
        id: changesets
        uses: changesets/action@v1
        with:
          publish: pnpm release
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
 		"build-esm": "tsc",
 		"build-cjs": "tsc -p tsconfig-cjs.json",
 		"fix-dist": "echo '{\"type\": \"module\"}' > dist/esm/package.json && echo '{\"type\": \"commonjs\"}' > dist/cjs/package.json",
-		"build": "run-s -l build-esm build-cjs fix-dist",
+		"build": "run-s -l cs-version build-esm build-cjs fix-dist",
 		"cs-version": "changeset version && pnpm version --json > versions.json",
 		"cs-publish": "changeset publish",
 		"release": "run-s -l clean cs-version build cs-publish",

